.github/workflows/release-publish-npm.yml

```diff
@@ -0,0 +1,46 @@
+name: Publish to NPM
+on:
+  release:
+    types: [published]
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '18'
+          registry-url: 'https://registry.npmjs.org'
+          
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: '1.21'
```

Comment:
@KilianBoute  we are using a newer go version. https://github.com/uug-ai/models/blob/main/go.mod#L3

Reply:
have a look here -> https://github.com/uug-ai/hub-pipeline-notification/blob/main/.github/workflows/release-create.yml#L26-L31